jaypeasee/sorting-hat-quiz
# Feature - content injection

#### What’s this PR do?
This PR creates a user experience out of the logic that was introduced after the last PR. Specifically it:
1. interpolates/injects data into the `Results` component to make it dynamic to the user's house.
2. Creates a form validation process in the `Name` Component
3. Adds some minimal styling to the entire app.

#### Where should the reviewer start?
1. Review `Result.js` to see the content injection
2. Review `Name.js` to see the form validation use of router based on onSubmit and conditions.
3. Review all style sheets to see how the app is styled.

#### How should this be manually tested?
1. Run `npm start`
2. Start the quiz.
3. Try to break the `Name` form to see how it behaves.
4. Proof-read the `Result` page to make sure data is interpolated correctly.

#### Any background context you want to provide?
This completes a user experience.


#### What are the relevant tickets?
* #19 
* #20
* #5 
* #3 

#### Screenshots (if appropriate):

![Screen Shot 2021-01-14 at 11 30 20 AM](https://user-images.githubusercontent.com/59029768/104633116-e93c7c00-565b-11eb-8236-8fdf59bd935d.png)
